### ISSUE TITLE
Doesn't handle psql casting without whitespace

### ISSUE THREAD

**Issue by kg4lod** [layerware/hugsql#30]
This is/should-be valid HugSQL because it is valid PSQL (works correctly in Yesql).

```
SELECT * FROM "the_table"
WHERE id = :key::bigint
```

But it will generate 

```
SELECT * FROM "the_table" WHERE id = $1
```

instead of 

```
SELECT * FROM "the_table" WHERE id = $1::bigint
```

The work-around is to add whitespace between the cast and the keyword, but it should support the concatenated format, no?

**Comment by csummers** [comment]
HugSQL tries to support Postgresql type casting with `::`, but this is a bit of an edge case when it is immediately preceded by a `:parameter` because of HugSQL's `:param-type:param-name` syntax.

HugSQL does support [escaping the colon](http://www.hugsql.org/#escaping-colon) with `\:`.
```
SELECT * FROM "the_table"
WHERE id = :key\:\:bigint
```

I'll keep looking at this and see if I can build in a work-around.

**Comment by csummers** [comment]
Support for this has been released in 0.4.5.  Please confirm it works for you.  Thanks for the report!
